From a6e78e9930d9eeac1dc493346cd90358161b9a07 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Tue, 13 Mar 2012 16:48:53 +0800
Subject: [PATCH 1190/2463] ENGR00177083-2 i.mx6: sabresd: move pfuse
 declaration to common.h

- Move mx6q_sabresd_init_pfuze100 to common.h
- Delete the declarations which are already in common.h

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c  |    6 ------
 arch/arm/plat-mxc/include/mach/common.h |    1 +
 2 files changed, 1 insertions(+), 6 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index b044171..e2bc0b9 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -112,18 +112,12 @@
 #define SABRESD_GPS_EN		IMX_GPIO_NR(3, 0)
 #define SABRESD_AUX_3V15_EN	IMX_GPIO_NR(6, 9)
 
-void __init early_console_setup(unsigned long base, struct clk *clk);
 static struct clk *sata_clk;
 static int mma8451_position = 3;
 static int mag3110_position;
 
 extern char *gp_reg_id;
 
-extern struct regulator *(*get_cpu_regulator)(void);
-extern void (*put_cpu_regulator)(void);
-extern void mx6_cpu_regulator_init(void);
-extern int __init mx6q_sabresd_init_pfuze100(u32 int_gpio);
-
 static iomux_v3_cfg_t mx6q_sabresd_pads[] = {
 	/* AUDMUX */
 	MX6Q_PAD_CSI0_DAT4__AUDMUX_AUD3_TXC,
diff --git a/arch/arm/plat-mxc/include/mach/common.h b/arch/arm/plat-mxc/include/mach/common.h
index b2cacfb..89bc884 100755
--- a/arch/arm/plat-mxc/include/mach/common.h
+++ b/arch/arm/plat-mxc/include/mach/common.h
@@ -92,4 +92,5 @@ extern int mxs_reset_block(void __iomem *);
 extern void early_console_setup(unsigned long base, struct clk *clk);
 extern void mx6_cpu_regulator_init(void);
 extern int mx6q_sabreauto_init_pfuze100(u32 int_gpio);
+extern int mx6q_sabresd_init_pfuze100(u32 int_gpio);
 #endif
-- 
1.7.7.4

